crates/bridge-exec/src/handler.rs

```diff
@@ -24,7 +24,7 @@ use crate::errors::{ExecError, ExecResult};
 /// The interval that the bridge duty exec functions will poll for
 /// bridge messages, such as public nonces and partial signatures.
 // TODO: this is hardcoded, maybe move this to a user-configurable Config
-pub const POLL_INTERVAL: Duration = Duration::from_millis(100);
+pub const POLL_INTERVAL: Duration = Duration::from_millis(1_000);
```

Comment:
Since we're in here we could try to make this configurable.

Reply:
Done in https://github.com/alpenlabs/strata/commit/13b7ab76de48ea14f986aeb16bcfd789f032b913.